src/app/footer/footer.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="container">
+  <footer class="d-flex flex-wrap justify-content-between align-items-center py-3 my-4 border-top">
+    <div class="col-md-4 d-flex align-items-center">
+      <a href="https://drive.google.com/file/d/16TeuCO5okzavcMZWuCmomBVv1a4ni70G/" target="_blank" class="mb-3 me-2 mb-md-0 text-muted text-decoration-none lh-1">OCH</a>
+    </div>
+    <ul class="nav col-md-4 justify-content-end list-unstyled d-flex">
+      <li class="ms-3">
+        <a class="text-muted" href="mailto:czeizinghedi@gmail.com">
+          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-google" viewBox="0 0 16 16">
```

Comment:
I suggest using bootstrap icons instead of directly pasting .svg images here.
see: https://icons.getbootstrap.com/#icon-font